griptape/artifacts/audio_artifact.py

```diff
@@ -24,3 +34,9 @@ def to_bytes(self) -> bytes:
 
     def to_text(self) -> str:
         return f"Audio, format: {self.format}, size: {len(self.value)} bytes"
+
+    def make_name(self) -> str:
+        entropy = "".join(random.choices(string.ascii_lowercase + string.digits, k=4))
+        fmt_time = time.strftime("%y%m%d%H%M%S", time.localtime())
```

Comment:
Out of scope for PR, but I'd like to see a util for the common logic between image/audio filename